lib/manager/npm/post-update/node-version.ts

```diff
@@ -29,26 +29,45 @@ function getPackageJsonConstraint(config: PostUpdateConfig): string | null {
 }
 
 export async function getNodeConstraint(
-  config: PostUpdateConfig
+  config: PostUpdateConfig,
+  allowUnstable = false
 ): Promise<string> | null {
   const { packageFile } = config;
   let constraint =
     (await getNodeFile(getSiblingFileName(packageFile, '.nvmrc'))) ||
     (await getNodeFile(getSiblingFileName(packageFile, '.node-version'))) ||
     getPackageJsonConstraint(config);
+  let lockfileVersion = 1;
+  try {
+    const lockFileName = getSiblingFileName(packageFile, 'package-lock.json');
+    lockfileVersion = JSON.parse(await readLocalFile(lockFileName, 'utf8'))
```

Comment:
`JSON.parse`  will throw for empty json, which we get if no lockfile. so we get an anoying debug message with a big stack trace.

SO we should at least check the return value from `readLocalFile` before try to parse it.

Reply:
I decided to log nothing. If there was a problem with this file it would have been logged earlier anyway